qa/workunits/rados/test_pool_quota.sh

```diff
@@ -12,7 +12,7 @@ done
 
 sleep 30
 
-rados -p $p put onemore /etc/passwd  && exit 1 || true
+rados -p $p put onemore /etc/passwd  &
```

Comment:
to make sure these background processes exit successfully, we need to save their pid and wait for them at the end of the test (e.g. PID1=$! after running the command, and wait $PID1 at the end). Plain wait (no args) doesn't check child process return values.